apis/filesystem/src/main/java/org/jclouds/filesystem/config/FilesystemBlobStoreContextModule.java

```diff
@@ -39,7 +41,12 @@
    protected void configure() {
       bind(BlobStore.class).to(LocalBlobStore.class);
       install(new BlobStoreObjectModule());
-      bind(ConsistencyModel.class).toInstance(ConsistencyModel.STRICT);
+      if (isWindows()) {
+         bind(ConsistencyModel.class).toInstance(ConsistencyModel.EVENTUAL);
+      }
+      else {
```

Comment:
[nit] Formatting (`else` on the same line?). We can do this during the merge, though - no need to update the PR, from my perspective.

@gaul: Your comment about regarding as eventually consistent - does this work for you now?